src/index.html

```diff
@@ -20,8 +20,504 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          class="logo"
```

Comment:
It would be a better BEM practice to assign this class based on the parent block (header)